Ryan's Wiki Changes

TheCoub opened the pull request:
Renamed the controls to the two tap system. Added Deconstruct as a control. Removed directional specific controls from the master table in order to create uniformity with what was already there.

TheCoub committed:
Renamed to the Two-Tap System and Added Deconstruct

Renamed the controls to the two tap system. Added Deconstruct as a control. Removed directional specific controls from the master table in order to create uniformity with what was already there.

TheCoub committed:
Deleted Contributors page

Deleted the Contributors page. it will be reworked and re-added in at a later date

JCharante commented:
Please revert `c060091` as you should not just delete the file and create a `pr`, you should just update it and when you do then create a `pr`.

Also, rename the file name for two tap system to two tap from three tap.

TheCoub committed:
Changed to New format

Added health, create a more usable summary of the uses, deleted the origins section.

TheCoub committed:
Changed to the new format

TheCoub committed:
Fixed a typo

TheCoub committed:
Converted to the new format

I also deleted some irrelevant information

TheCoub committed:
Fixed a typo

I didn't bother proofreading.

TheCoub committed:
Changed to the new format

TheCoub committed:
Marked for conversion

jyan-zarate commented on wiki/Controls_(The_Two_Tap_System).md:
```diff
@@ -1,6 +1,6 @@
+# Controls in And/Ore work on what we call "The Double-Tap System." 
 
-The Triple-Tap System involves pressing a "mode" key which determines which mode you are in, and then a modifier key to change the exact actions. The various modes and their mode keys are: 
+The Two-Tap System involves pressing a "mode" key which determines which mode you are in, and then a modifier key to change the exact actions. The various modes and their mode keys are: 
```
The `Two-Tap System` sounds less smooth than `Double-Tap System`

TheCoub replied:
Really? I find Alleteration makes it smooth. Its also less sylablles.

jyan-zarate commented on wiki/door.md:
```diff
@@ -5,25 +5,17 @@ Door
+- Ryan Courbron
 
-##### Page last edited on 10/14/16
+##### Page last edited on 12/8/2016 (Version 1.5, The Benovolent Bearded Dragon)
```
I really like how you added the name of the release, I love it!

Liham committed:
Update spiketrap.md

Liham committed:
Update spiketrap.md

Liham committed:
Update spiketrap.md

Liham committed:
Update spiketrap.md

Liham committed:
Update spiketrap.md

Liham committed:
Update spiketrap.md

Liham committed:
Update spiketrap.md

Liham committed:
Update spiketrap.md

jyan-zarate commented on wiki/spiketrap.md:
```diff
@@ -6,13 +6,15 @@ Spike Trap
 ============
-- Graphyte
-##### Page last edited on 10/07/16
+- Liam Connor-Schaefer
```
Wait, why'd you get rid of Graphyte's name under Contributors?
